esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6118,7 +6130,329 @@ bool CWsDfuEx::onDFUFileAccess(IEspContext &context, IEspDFUFileAccessRequest &r
             userDesc.setown(createUserDescriptor());
             userDesc->set(userID.str(), context.queryPassword(), context.querySignature());
         }
-        getFileAccess(context, userDesc, accessType, req, resp);
+        getFileAccess(context, userDesc, accessType, requestBase, resp.updateAccessInfo());
+    }
+    catch (IException *e)
+    {
+        FORWARDEXCEPTION(context, e,  ECLWATCH_INTERNAL_ERROR);
+    }
+    return true;
```

Comment:
is there any reason onDFUFileAccess has a boolean result? given it always returns true..

Reply:
I do not know the history.